CONTRIBUTING.md

```diff
@@ -10,17 +10,29 @@ Happy coding :dancer:
 
 Check out our SDK documentation for [requirements](https://docs.momentohq.com/sdks/php#requirements)!
 
-## Build Docker Image
+## Setup
+There are two ways to setup your development environment:
+
+
+## Option 1: Build Docker Image
 
 You will likely need to run this only once:
 
 ```bash
 ./dev-docker-build.sh
 ```
 
-## Run Integration Tests
+### Run Integration Tests
 
 ```bash
 export MOMENTO_API_KEY=<YOUR_API_KEY>
 ./dev-run-integration-tests.sh
 ```
+
+## Option 2: Running a Dev Container in IntelliJ IDEA
```

Comment:
We might want to make this more general or add a section on using the dev container in vscode once someone figures out how to do that.

Reply:
I'll try running this in vscode and document those steps as well!